src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -337,7 +337,23 @@ public static Scheduler onComputationScheduler(@NonNull Scheduler defaultSchedul
 
     /**
      * Called when an undeliverable error occurs.
+     * <p>
+     * Undeliverable errors are those {@code Observer.onError()} invocations that are not allowed happen to
```

Comment:
Nit: 
.. that are not allowed **to** happen to ...
?

Reply:
Updated.